app/Http/Controllers/MappingController.php

```diff
@@ -165,4 +170,56 @@ public function destroy(Webhook $webhook, Mapping $mapping)
             ]);
         }
     }
+
+    public function import(Webhook $webhook, MappingImportRequest $request)
+    {
+        DB::beginTransaction();
+        try {
+            $json = file_get_contents($request->file);
+            $getKeyValue = $this->mappingRepository->getKey($webhook);
+            $data = json_decode(file_get_contents($request->file), true);
+            $newData = [];
+            foreach($data as $key => $item) {
+                if (!in_array($item['key'], $getKeyValue)) {
+                    array_push($newData, array_merge($item, ["webhook_id" => $webhook->id]));
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/MappingController.php</br>**Line**: 184</br>String "webhook_id" does not require double quotes; use single quotes instead